geoapps-assets/uijson/block_model_creation.ui.json

```diff
@@ -1,18 +1,18 @@
 {
+    "version": "0.11.0-alpha.1",
```

Comment:
Will need to udpate those to 0.12.0. We can punt it to another issue too

Reply:
yes, other issue, I am just fixing the env and build here
(plus some auto-formatting)